BEPs/BEP-336.md

```diff
@@ -0,0 +1,501 @@
+<pre>
+  BEP: 336
+  Title: Shard Blob Transactions
+  Status: Draft
+  Type: Standards
+  Created: 2023-12-04
+</pre>
+
+# BEP-336: Shard Blob Transactions
+
+- [BEP-336: Shard Blob Transactions](#bep-336-shard-blob-transactions)
+  - [1.  Summary](#1-summary)
+  - [2.  Abstract](#2-abstract)
+  - [3.  Status](#3-status)
+  - [4.  Motivation](#4-motivation)
+  - [5.  Specification](#5-specification)
+  - [6.  Rationale](#6-rationale)
+  - [7. Backwards Compatibility](#7-backwards-compatibility)
+  - [8. Security Considerations](#8-security-considerations)
+  - [9. Life of a Blob Transaction](#9-life-of-a-blob-transaction)
+  - [9.  Life of a Blob Transaction](#9-life-of-a-blob-transaction)
+    - [9.1 Blob pool](#91-blob-pool)
+    - [9.2 Verification / Filtering](#92-verification--filtering)
+    - [9.3 Consensus process: Generation & Storage of Sidecars](#93-consensus-process-generation--storage-of-sidecars)
+    - [9.4 Propagation of Sidecars among peers](#94-propagation-of-sidecars-among-peers)
+    - [9.5 Fetching of Sidecars when a node starts](#95-fetching-of-sidecars-when-a-node-starts)
+  - [10. Changes Specifc to BSC](#10-changes-specifc-to-bsc)
+    - [10.1 Verification](#101-verification)
+    - [10.2 Sidecar Propagation / P2P network](#102-sidecar-propagation--p2p-network)
+    - [10.3 Sidecar Fetching/Downloading](#103-sidecar-fetchingdownloading)
+  - [11. License](#11-license)  
+
+## 1. Summary
+
+This BEP proposes introduction of blob-carrying transactions which may contain large amount of data.
+
+## 2. Abstract 
+
+Introduce a new transaction format for “blob-carrying transactions” which contain a large amount of data that cannot be accessed by EVM execution, but whose commitment can be accessed. The format is intended to be fully compatible with the format that will be used in full sharding.
+
+## 3. Status 
+
+Work in progress. 
+A PoC has been created in this branch: https://github.com/bnb-chain/bsc/tree/4844 
+
+## 4. Motivation 
+
+Rollups such as opBNB are in the short and medium term, and possibly in the long term, the only trustless scaling solution for Ethereum & BSC. Transaction fees on L1 have been very high for months and there is greater urgency in doing anything required to help facilitate an ecosystem-wide move to rollups. Rollups are significantly reducing fees for many Ethereum users: Optimism and Arbitrum frequently provide fees that are ~3-8x lower than the Ethereum base layer itself, and ZK rollups, which have better data compression and can avoid including signatures, have fees ~40-100x lower than the base layer.
```

Comment:
`Transaction fees on L1 have been very high for months`
It is not true for BSC.

Reply:
We may refer the EIP-4844, but need some adjustment to reflect BSC's status.